dist/bin/zincati-update-now

```diff
@@ -23,4 +23,4 @@ systemctl daemon-reload
 systemctl restart zincati --no-block
 
 echo "INFO: Streaming Zincati and RPM-OSTree logs..." >&2
-exec journalctl -f -u zincati -u rpm-ostreed
+exec journalctl -f -u zincati -u rpm-ostreed --since now
```

Comment:
I think `--invocation=0` would be even nicer, as `--since-now` is leaving out on any messages between the restart and the execution of this. 
But that's being picky as it's an extremely short race :smile: 

And `-I` was just introduced in `v257` which isn't in F41. (it's in branched though so if this lands in the same release as the beta rebase that would be OK).

Reply:
nice - didn't know about `-I`. but yeah this is going to go into F41 too so let's not pick it up and use it yet